libbuteosyncfw/profile/SyncSchedule.h

```diff
@@ -170,13 +168,13 @@ class SyncSchedule
      * \return True if schedule is controlled by a external process. External process will control the sync,
      * buteo schedule is disabled in this case.
      */
-    bool externalScheduleEnabled() const;
+    bool alwaysOnEnabled() const;
 
     /*! \brief Sets if schedule is controlled by a external process.
      *
      * \param aEnabled Specify if schedule is contolled by a external process. If set to true, buteo schedule will be set to disabled.
      */
-    void setExternalScheduleEnabled(bool aEnabled);
+    void setAlwaysOnEnabled(bool aEnabled);
```

Comment:
Ok, I can change the names again.
As for the schedule it makes more sense to me to have this there, since it does affect the schedule and makes things more clear since when this property is on, schedule enabled should be false. Also same property is needed anyway inside the rush schedule, so if I change it, related property will be spread around.